catalyst/dl/experiment/config.py

```diff
@@ -319,6 +322,66 @@ def get_scheduler(self, stage: str, optimizer: Optimizer) -> Scheduler:
         )
         return scheduler
 
+    @staticmethod
+    def _get_transform(**params) -> Callable:
+        key_value_flag = params.pop("_key_value", False)
+
+        if key_value_flag:
+            transforms_composition = {
+                key: ConfigExperiment._get_transform(**params_)
+                for key, params_ in params.items()
+            }
+
+            transform = {
+                key: Augmentor(
+                    dict_key=key,
+                    augment_fn=transform,
+                    input_key=key,
+                    output_key=key,
+                )
+                for key, transform in transforms_composition.items()
+            }
+        else:
+            if "transforms" in params:
+                transforms_composition = [
+                    ConfigExperiment._get_transform(**transform_params)
+                    for transform_params in params["transforms"]
+                ]
+                params.update(transforms=transforms_composition)
+
+            transform = TRANSFORMS.get_from_params(**params)
+
+        return transform
+
+    def get_transforms(self, stage: str = None, mode: str = None) -> Callable:
```

Comment:
I am not sure it's a `mode`, from your examples it looks like `loader_{name}`/`dataset_{name}`/ etc?

Reply:
yes, what name instead of `mode` could you suggest to use to prevent confusion?

Reply:
what do you think about `loader` (as we are using transforms for loaders)